src/users/users.service.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  ConflictException,
+  Injectable,
+  NotFoundException,
+} from '@nestjs/common'
+import { CreateUserDto } from './dto/create-user.dto'
+import { UpdateUserDto } from './dto/update-user.dto'
+import { UserRole } from 'src/util/role.enum'
+import { InjectRepository } from '@nestjs/typeorm'
+import { Repository } from 'typeorm'
+import { User } from './entities/user.entity'
+import * as bcrypt from 'bcrypt'
+
+@Injectable()
+export class UsersService {
+  constructor(
+    @InjectRepository(User)
+    private usersRepository: Repository<User>,
+  ) {}
+
+  findAll(): Promise<User[]> {
+    return this.usersRepository.find()
+  }
+
+  async create(createUserDto: CreateUserDto): Promise<User> {
```

Comment:
this function should be renamed to `signup` and then moved to the `auth.service.ts`